.github/workflows/deploy-playground.yml

```diff
@@ -61,7 +61,7 @@ jobs:
 
       - name: Deploy connect next playground to Vercel
         run: |
-          cd playground/connect-next
```

Comment:
we now follow vercel's advice for monorepos: https://vercel.com/docs/monorepos (doing everything from the root directory)